### PR TITLE
Fixed minimum required PHP version in readme.txt file of future 8.0 version

### PR DESCRIPTION
**Pull request by DAnn2012**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Fixed minimum required PHP version in readme.txt file of future 8.0 version

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.
2.
3.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [x] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement

#### Message 

Fixed minimum required PHP version in readme.txt file of future 8.0 version

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/readme.txt [hunk 1]
```diff
@@ -136,7 +136,7 @@ Check out [Frequently Asked Questions](https://docs.woocommerce.com/document/fre
 
 = Minimum Requirements =
 
-* PHP 7.2 or greater is recommended
+* PHP 7.3 or greater is recommended
 * MySQL 5.6 or greater is recommended
```
Comment (Konamiman) [inline plugins/woocommerce/readme.txt]: This aligns better with our current recommendations:

```suggestion
* PHP 7.3 or greater is required (PHP 8.0 or greater is recommended)
* MySQL 5.6 or greater, OR MariaDB version 10.1 or greater, is required
```

**Review by Konamiman — CHANGES_REQUESTED** [review]

**Comment by Konamiman** [comment]
@DAnn2012 could you please add a changelog file to the pull request? You can do si by running `pnpm --filter=woocommerce run changelog add`. Thank you!

**Comment by DAnn2012** [comment]
Hi @Konamiman 

I made the required changes and added the changelog file manually here: https://github.com/DAnn2012/woocommerce/commit/8c53749908618a7f562028aa20d17ab9d75ab69d

Is that okay too?

Thank you.

**Comment by Konamiman** [comment]
Hi @DAnn2012, thank you but something weird is happening: although I can see the commit in which the changelog file is added, that commit doesn't appear as being part of the pull request. How have you created/pushed the commit?

**Comment by DAnn2012** [comment]
Hi @Konamiman 

I clarify that I don't know and don't use CLI, I added the file directly in my `patch-3` branch using the **Add file -> Create new file** button

**Comment by Konamiman** [comment]
@DAnn2012 That should have worked but for some reason didn't. Not sure what really happened.

Anyway, it's ok: I was able to remove your commit and add a new one myself, so I'll approve this.

**Review by Konamiman — APPROVED** [review]

**Pull request merged**